File: Lee.EMS/8.1.0/vue/src/store/entities/sysmodule.ts

/*-------------------------------------------------------
// Copyright (C) 2020 Lic
//
// 功能描述:模块关联  实体对象
//
//
// 创建标识: Lee -- 2023-07-30 15:48:53
//
//------------------------------------------------------*/
 
import Entity from './entity'
// 模块关联 
export default class SysModule extends Entity<number>{
    parentId:number;
    name:string;
    icon:string;
    sort:number;
    visible:boolean;
    pageRoute:string;
    describe:string;
    constructor(){
        super();
        this.visible=true;
    }
}

File: Lee.EMS/8.1.0/aspnet-core/src/Lee.EMS.Core/SYS/SysModule.cs
using Abp.Domain.Entities.Auditing;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics.CodeAnalysis;

namespace Lee.EMS.SYS
{
    [Table("Sys_Module")]
    public class SysModule : AuditedEntity<long>
    {
        /// <summary>
        /// 
        /// </summary>
        [Column("ParentId")]
        public long ParentId { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Name"), MaxLength(100), NotNull]
        public string Name { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Icon"), MaxLength(200)]
        public string Icon { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Sort")]
        public long Sort { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Visible")]
        public bool Visible { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("PageRoute"), MaxLength(100)]
        public string PageRoute { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Column("Describe"), MaxLength(500)]
        public string Describe { get; set; }
    }
}
